### ISSUE TITLE
Why is there multiple classification results for the same bounding box?

### ISSUE THREAD

**Issue by CynthiaLai** [AlexeyAB/darknet#3206]
I am wondering if the same bounding box would have multiple classification results?
Used 'test" command like that :
![未命名](https://user-images.githubusercontent.com/12018943/58144349-ab1d6800-7c80-11e9-96b1-9ce758cc135e.png)

**Comment by AlexeyAB** [comment]
Yes, one bbox can predict several classes simultaneously

https://arxiv.org/pdf/1804.02767v1.pdf

> 2.2. Class Prediction
> Each box predicts the classes the bounding box may contain using multilabel classification. We do not use a softmax
> as we have found it is unnecessary for good performance,
> instead we simply use independent logistic classifiers. During training we use binary cross-entropy loss for the class
> predictions
